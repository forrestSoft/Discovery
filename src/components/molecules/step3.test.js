import React from 'react'
import {shallow, mount} from 'enzyme'
import shallowToJson from 'enzyme-to-json'

import Step3 from './step3'

it('render matches snapshot', () => {
  const mock = jest.fn().mockReturnValue('onClick')
  const step3 = shallowToJson(shallow(
  	<Step3
      handleChange={mock}
      data={
        {
          firstName: '',
          firstNameValid: false,
          lastName: '',
          lastNameValid: false
        }
      }
    />
  )) 
  expect(step3).toMatchSnapshot();
});