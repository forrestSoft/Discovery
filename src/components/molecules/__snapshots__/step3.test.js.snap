// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render matches snapshot 1`] = `
<Fragment>
  <Text
    Type="h1"
    text="Thank You For Signing Up!"
    type="h2"
  />
  <Text
    Type="h1"
    text="Look out for the latest news on your favorite shows."
    type="h3"
  />
</Fragment>
`;
